compiler/rustc_trait_selection/src/traits/wf.rs

```diff
@@ -689,6 +696,7 @@ impl<'tcx> WfPredicates<'tcx> {
         &mut self,
         def_id: DefId,
         substs: SubstsRef<'tcx>,
+        remap_constness: bool,
```

Comment:
Rename this function, make it private and create two wrappers around it. One with the original name, passing false, and one with a trailing with_constness `` or similar